Portfolio/index.css

```diff
@@ -1 +1,67 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;
+          margin: 0%;
+    padding: 0%;
+}
+.cont1{
+    background-color: #90A4AE;
+    padding: 10px;
+  
+}
+nav ul{
+    padding: 0px;
+    margin: 0px;
+}
+nav li{
+    list-style-type: none;
+}
+.button{
+    border-radius: 20px;
+    padding: 10px;
+    border-color: #515050;
+}
+img{
+    border-radius: 50%;
+}
+form{

```

Comment:
Consider adding more specific styling properties or values to the .contact class for better differentiation